Assets/Scripts/Player/PlayerStats.cs

```diff
@@ -22,11 +24,16 @@ public class PlayerStats : MonoBehaviour
         [Header("Throw Speed")] [SerializeField]
         public float ThrowVelocity;
 
+        [FormerlySerializedAs("TeamAffiliation")] [Header("Team Affiliation")] [SerializeField]
```

Comment:
the `FormerlySerializedAs` and `SerializeField` attributes can be removed here without any implications